pdal/DynamicLibrary.cpp

```diff
@@ -71,6 +72,16 @@ void DynamicLibrary::clear()
     m_handle = NULL;
 }
 
+#ifdef _WIN32
+std::wstring toNative(std::string const& in)
```

Comment:
Even if this is the only place this is used, it should probably be in PDALUtils for symmetry and so that people know where to find it.

What's the with "static thread local" comment? Why would it matter?

Reply:
I just copied it from PDALUtils::FileUtil